tutorials/lesson2-integers/integers.template.qnt

```diff
@@ -0,0 +1,337 @@
+/*
+ If you do not know how to start this lesson, do one of the following:
+
+ 1. Run a Codetour tutorial as follows:
+    - Open Visual Studio Code.
+    - Install the Quint and Codetour extensions.
+    - Choose the tour "Lesson 2 - Integers".
+    
+ 2. Read the accompanying markdown file.
+ */
+/*!
+  <tutorial>
+    <title>Lesson 2 - Integers</title>
+    <steps>
+ !*/
+module integersLesson {
+    /*!
+      <step>
+        <title>Introduction</title>
+        <description>
+**Scores to earn with secret codes: 10**
+
+This lesson teaches you the basics of operations over integers.
+If you have programming experience, you know most of these operators. 
+So it should not take you long to finish this lesson.
+Do not skip this lesson, as some of the operators may still surprise you.
+        </description>
+        <file>$file</file>
+        <line>$line</line>
+      </step>
+    !*/
+
+    // 0 is an integer literal
+    pure val int0 = 0
+
+    // 1 is an integer literal
+    pure val int1 = 2
+
+    // 2 is an integer literal
+    pure val negative1 = -3
+    /*!
+      <step>
+        <file>$file</file>
+        <line>$line</line>
+        <title>Integer literals</title>
+        <description>
+Integer literals are written using the standard syntax:
+0, 1, -1, 2, -2, ..., 314159265358979323846264338327950288419716939937510.
+
+It is important to understand that Quint integers are big integers.
+There is neither a minimum integer, nor there is a maximum integer.
+
+It's quite easy to express the standard 32-bit, 64-bit, 128-bit, and 512-bit
+integers with Quint integers. We will cover this in a follow up tutorial.
+
+We omit several integer operators that produce sets. These operators are covered in
+the tutorial on sets.
+        </description>
+      </step>
+     !*/
+
+    // i^j is the integer exponentiation, that is,
+    // `i` multiplied by itself `j - 1` times
+    pure def myPow(i, j) = i^j 
```

Comment:
In the markdown version, this is shown after the examples, while it seems like it should appear before them.

Reply:
Yes, I have found this way more natural. In Codetour, the explanation looks better below the code, whereas in Markdown the code looks better as an example.